modules/calendar/calendar.py

```diff
@@ -39,7 +39,7 @@ def get_calendar(
 		if group_id:
 			groups = groups or Group.get_groups()
 			# get the group specified by the user given the group id
-			group = one(groups, lambda group: group.group_id == group_id)
+			group = one(group for group in groups if group.id == group_id)
```

Comment:
Wasn't the idea behind changing this in order to make it so it can return a result before filtering the entire list?

Reply:
its most often used without a filter, and this looks much neater